yktracec/src/jitmodbuilder.cc

```diff
@@ -459,6 +470,9 @@ class JITModBuilder {
   // compilation.
   CallStack CallStack;
 
+  // Flag showing whether this trace is a normal trace or a side trace.
```

Comment:
Better something like "Set to true for a sidetrace or false for a normal trace"?

Reply:
Fixed in 8933af0be67